### PR TITLE
docs: Update Typography docs

### PR DESCRIPTION
**Pull request by rodrigoeidelvein**
## Motivations

Updating Typography docs, it's web and mobile.

### Added

- 

### Changed

- Update Typography docs

### Deprecated

- 

### Removed

- 

### Fixed

- 

### Security

- 

## Testing

---

[In Atlantis we use Github's built in pull request reviews](https://help.github.com/en/articles/about-pull-request-reviews).

Random photo of Atlantis

### CODE REVIEW

File: docs/components/Typography/Typography.stories.mdx [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+
+Typography is the component that powers our Heading and Text components “under
+the hood”. You should first look to use
+[Heading](../?path=/docs/components-text-and-typography-heading--heading) and
+[Text](../?path=/docs/components-text-and-typography-text-docs--page) before you
+reach for Typography.
+
+</Banner>
+
+## Font size
```
Comment (taylorvnoj) [inline docs/components/Typography/Typography.stories.mdx:21]: Maybe can we have a sentence here linking to the Design Typography section where we have the web and mobile size tokens?

Comment (taylorvnoj) [reply docs/components/Typography/Typography.stories.mdx:21]: should be (../?path=/docs/design-typography--page#font-size)

**Review by taylorvnoj — CHANGES_REQUESTED** [review]
Looks good! Just two suggestions

File: docs/components/Typography/Typography.stories.mdx (hunk 1 above)
Comment (rodrigoeidelvein) [reply docs/components/Typography/Typography.stories.mdx:21]: Added it! good idea

**Review by taylorvnoj — APPROVED** [review]
LGTM!

**Pull request merged**